mgmtsystem/ec2.py

```diff
@@ -537,3 +540,50 @@ def delete_loadbalancer(self, loadbalancer):
 
     def get_all_unused_network_interfaces(self):
         return [eni for eni in self.api.get_all_network_interfaces() if eni.status == "available"]
+
+    def import_image(self, s3bucket, s3key, format="vhd", description=None):
+        self.logger.info(" Importing image {} from {} bucket with description {} in {} started "
+                         "successfully."
+                         "format".format(s3key, s3bucket, description, format))
+        try:
+            result = self.ec2_connection.import_image(DiskContainers=[
+                {
+                    'Description': description if description is not None else s3key,
+                    'Format': format,
+                    'UserBucket': {
+                        'S3Bucket': s3bucket,
+                        'S3Key': s3key
+                    }
+                }
+            ])
+            image_id = result.get("ImageID")
+            return image_id
```

Comment:
I found that if import is not completed then I will not be able to get imageid, so I reworked is image import completed to get_image_id_if_import_completed.
Basically steps:
1) you first upload it and you have the name
2) You add the name to image_import and you get image_import_task_id
3) You check for completion of import task by image_import_task_id. If it was already completed then you get image_id
4) Now you can copy images when you have image_id.